jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/postgresql/PostgreSQLSqlDialectIT.java

```diff
@@ -278,6 +278,27 @@ void fullOuterJoin() throws SQLException {
         assertFalse(result.next());
     }
 
+    @Test
+    void rightJoinWithComplexCondition() throws SQLException {
```

Comment:
Add the prefix `test` to test methods please,

Reply:
done